src/block/tab-content/edit.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames'
+import { InspectorTabs, ColumnInnerBlocks } from '~stackable/components'
+import {
+	BlockDiv,
+	useGeneratedCss,
+	Alignment,
+	Advanced,
+	CustomCSS,
+	Responsive,
+	CustomAttributes,
+	EffectsAnimations,
+	ConditionalDisplay,
+	Separator,
+	Transform,
+} from '~stackable/block-components'
+import {
+	withBlockAttributeContext,
+	withBlockWrapperIsHovered,
+	withQueryLoopContext,
+} from '~stackable/higher-order'
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose'
+import { __ } from '@wordpress/i18n'
+
+const ALLOWED_INNER_BLOCKS = [ 'stackable/column' ]
+const TEMPLATE = [
+
+]
+
+const Edit = props => {
+	const {
+		className,
+		isSelected,
+		context,
+	} = props
+
+	const tabCount = context[ 'stackable/tabCount' ]
+
+	useGeneratedCss( props.attributes )
+
+	const [ columnProviderValue, columnTooltipClass ] = ColumnInnerBlocks.useContext()
+
+	const blockClassNames = classnames( [
+		className,
+		columnTooltipClass,
+		'stk-block-tab-content',
+	] )
+
+	return (
+		<>
+			{ isSelected && (
+				<>
+					<InspectorTabs />
+					<Alignment.InspectorControls hasRowAlignment={ true } />
+					<BlockDiv.InspectorControls />
+					<Separator.InspectorControls />
```

Comment:
Go through all of the Block Components here and see whether you'll need them for the Tab Content block. One thing for sure is you won't need the `Separator` and `Conditional Display` here. Adjust the `save` and `style` parts accordingly.

Also go through the Block Components of the Tab Labels and Tabs block and check if they're relevant.